test/favorite.test.js

```diff
@@ -0,0 +1,147 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../server/app';
+
+
+chai.use(chaiHttp);
+
+const { expect } = chai;
+
+let token;
+
+
+describe('FAVORITES', () => {
+  before((done) => {
+    chai.request(app)
+      .post('/api/users/signin')
+      .send({
+        email: 'ngozinwali@gmail.com',
+        password: 'ngobest'
+      })
+      .end((err, res) => {
+        token = res.body.Token;
+        done();
+      });
+  });
+  it('should get all users favorite recipes', (done) => {
+    chai.request(app)
+      .get('/api/users/1/recipes')
+      .send({ token })
+      .end((err, res) => {
+        expect(res).to.have.status(200);
+        expect(res).to.be.json;
+        expect(res.body).to.have.property('Favorites');
```

Comment:
Testing for just the property `Favorites` might not be enough here. Assert for some of the expected data.